pkg/build/util/util.go

```diff
@@ -9,8 +9,12 @@ import (
 	"github.com/openshift/origin/pkg/util/namer"
 )
 
-// BuildPodSuffix is the suffix used to append to a build pod name given a build name
-const BuildPodSuffix = "build"
+const (
+	// BuildPodSuffix is the suffix used to append to a build pod name given a build name
+	BuildPodSuffix = "build"
+	// NoBuildLogsMessage reports that no build logs are available
+	NoBuildLogsMessage = "No logs are available"
```

Comment:
These constants don't seem to belong here - they aren't generic?

On Jun 29, 2015, at 10:24 AM, Ben Parees <notifications@github.com> wrote:

In pkg/build/util/util.go
<https://github.com/openshift/origin/pull/3449#discussion_r33468586>:

> @@ -9,8 +9,12 @@ import (
>  	"github.com/openshift/origin/pkg/util/namer"
>  )
>
> -// BuildPodSuffix is the suffix used to append to a build pod name given a build name
> -const BuildPodSuffix = "build"
> +const (
> +	// BuildPodSuffix is the suffix used to append to a build pod name given a build name
> +	BuildPodSuffix = "build"
> +	// NoBuildLogsMessage reports that no build logs are available
> +	NoBuildLogsMessage = "No logs are available"

period at the end, to align it with the rest of the message you're
appending this to.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3449/files#r33468586>.